client/src/modules/user/model/actions.js

```diff
@@ -149,3 +149,6 @@ export const getAccountDetails = token =>
 
 export const deleteAccount = (email, password) =>
   deleteData('/auth', { email, password });
+
+export const saveEmailNotificationSettings = settings =>
+  postData('/auth/email-notification-settings', settings);
```

Comment:
We always displayed notifications in actions (excluding forms, where we wanted to display errors inline)?